core/src/test/scala/unit/kafka/controller/ControllerIntegrationTest.scala

```diff
@@ -451,7 +451,7 @@ class ControllerIntegrationTest extends ZooKeeperTestHarness {
   }
 
   private def waitUntilControllerEpoch(epoch: Int, message: String): Unit = {
-    TestUtils.waitUntilTrue(() => zkClient.getControllerEpoch.get._1 == epoch, message)
+    TestUtils.waitUntilTrue(() => zkClient.getControllerEpoch == Some(epoch, AnyRef) , message)
```

Comment:
Why is `AnyRef` being passed as a parameter?

Reply:
`getControllerEpoch` returns a tuple of `Option[(Int, Stat)]`. Can we compare them in another way?

Reply:
That's true. I apparently haven't run the tests after that change ... embarrassing 

I updated the PR now